packages/generator-single-spa/src/root-config/generator-root-config.js

```diff
@@ -42,35 +42,36 @@ module.exports = class SingleSpaRootConfigGenerator extends Generator {
         type: "input",
         name: "orgName",
         message: "Organization name (use lowercase and dashes)",
+        default: "react-mf"
```

Comment:
We shouldn't have the organization name default to react-mf, since that's the name of the demo application.

```suggestion
        default: "org-name"
```